### PR TITLE
Reader Reduxification: adding in new tags action-types

### PR DESCRIPTION
**Pull request by samouri**
I've decided to split up https://github.com/Automattic/wp-calypso/pull/8589 into a series of pretty small PRs.  The first one is tiny!

Just the new action-types for migrating tags

Should be entirely dead code for now.  And the next 4 PRs will utilize them
- ~~request_tag~~, 
- request_tags,
-  follow_tag, 
- unfollow_tag

### CODE REVIEW

File: client/state/action-types.js [hunk 1]
```diff
@@ -442,6 +442,14 @@ export const READER_LIST_UPDATE_DESCRIPTION = 'READER_LIST_UPDATE_DESCRIPTION';
 export const READER_LIST_UPDATE_FAILURE = 'READER_LIST_UPDATE_FAILURE';
 export const READER_LIST_UPDATE_SUCCESS = 'READER_LIST_UPDATE_SUCCESS';
 export const READER_LIST_UPDATE_TITLE = 'READER_LIST_UPDATE_TITLE';
+export const READER_FETCH_TAG_REQUEST = 'READER_FETCH_TAG_REQUEST';
```
Comment (blowery) [inline client/state/action-types.js]: maybe READER_TAG_REQUEST?

File: client/state/action-types.js [hunk 2]
```diff
@@ -442,6 +442,14 @@ export const READER_LIST_UPDATE_DESCRIPTION = 'READER_LIST_UPDATE_DESCRIPTION';
...
+export const READER_FETCH_TAG_RECEIVE = 'READER_FETCH_TAG_REQUEST_RECEIVE';
```
Comment (blowery) [inline client/state/action-types.js]: maybe READER_TAG_RECEIVE?

File: client/state/action-types.js [hunk 3]
```diff
@@ -442,6 +442,14 @@ export const READER_LIST_UPDATE_DESCRIPTION = 'READER_LIST_UPDATE_DESCRIPTION';
...
+export const READER_FETCH_TAGS_REQUEST = 'READER_FETCH_TAGS_REQUEST';
```
Comment (blowery) [inline client/state/action-types.js]: maybe READER_TAG_SUBSCRIPTIONS_REQUEST?

Comment (samouri) [inline client/state/action-types.js]: hmm.  I don't want to have such different naming for two apis that return the same shape (one is tag other is tags). Looking more at it, they are even the same URL (one is /read/tags, other is /read/tags/{tagSlug}).  

I'm gonna reduce it down to one action type with a ternary to decide the path!

**Pull request merged**